riscv/csrs.h

```diff
@@ -6,8 +6,12 @@
 #include "encoding.h"
 // For reg_t:
 #include "decode.h"
+// For std::map
+#include <map>
```

Comment:
I guess this should be `unordered_map`? I can't see `std::map` used below, and this fails compile for me because `unordered_map` is undefined.

Reply:
Yes indeed -- will push an update along with resolving other rebase artifacts (like the bitmap features in isaparser.cc).